moped-editor/src/utils/test/dateAndTime.test.js

```diff
@@ -54,3 +55,39 @@ describe("makeFullTimeFromTimeStampTZ()", () => {
     expect(formattedTime).toBe("11:14:45 AM");
   });
 });
+
+describe("formatRelativeDate()", () => {
+  it("formats millesconds ago to a human description relative to now", () => {
+    const nowMills = new Date().getTime();
+    const twoSecondsAgoMills = nowMills - 1000 * 2;
+    const oneMinute20sAgoMills = nowMills - 1000 * 70;
+    const twoMinutesAgoMills = nowMills - 1000 * 60 * 2;
+    const fiftyNineMinutesAgo = nowMills - 1000 * 60 * 59;
+    const twoHoursAgoMills = nowMills - 1000 * 60 * 60 * 2;
+    const twoDaysAgoMills = nowMills - 1000 * 60 * 60 * 2 * 24;
+    const twoMonthsAgoMills = nowMills - 1000 * 60 * 60 * 2 * 24 * 60;
```

Comment:
I always enjoy seeing patterns like this; these are magic numbers that I can get behind. ✨ I like to turn off linting rules for chunks like this and line them all up -- it is totally useless but the columns look so organized 🤩